static/usr/lib/systemd/system/proc-sys-fs-binfmt_misc.mount.d/wait-modules.conf

```diff
@@ -0,0 +1,2 @@
+[Unit]
```

Comment:
If this is a temporary fix, I think we should add it as comment in the file.

Reply:
I've added comments to the two files now